spec/services/grant_code_exchanger_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
+
+RSpec.describe GrantCodeExchanger do
+  describe '#run' do
+    it 'saves the access_token and refresh_token for the sandwich' do
```

Comment:
I was going to talk about `build_stubbed` here, but yeah - if we want to keep this simple, FactoryBot is probably not a good thing to introduce.

Reply:
yeah as I write more tests I keep thinking about adding it...maybe in a PR or 2 it will seem obvious that we should. for now created seems OK